onset_maproom/calc.py

```diff
@@ -247,20 +400,59 @@ def seasonal_onset_date(
     dry_spell_search,
     time_coord="T",
 ):
-    """Function reproducing `Ingrid onsetDate function <http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate>`_
-    combining a function that groups data by season
-    and a function that search for an onset date
-
-    Parameters
-    ----------
-    daily_rain : DataArray
-                 Array of daily rainfall totals.
-    search_start_day : int
-                 The day part (1-31) of the date to start scanning for onset date.
-    search_start_month : int
-                 The month part (1-12) of the date to start scanning for onset date.
-    """
+    """ Computes yearly seasonal onset dates from daily soil moisture data.
+
+        Function reproducing Ingrid onsetDate function 
+        '<http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate>`_
+        
+        Combining a function that groups data by season and a function that searches for an onset date
+
+        Computes yearly dates by utilizing groupby function to group data by season 
+        and onset_date function to calculate cessation date for each season.
+
+        Parameters
+        ----------
+        daily_rain : DataArray
+                    Array of daily rainfall values.
+        search_start_day : int
+                    The day part (1-31) of the date to start scanning for onset date.
+        search_start_month : int
+                    The month part (1-12) of the date to start scanning for onset date.
```

Comment:
would be nice to have same phrasing here than for daily_tobegroupedby (and other seasonal_ functions)

Reply:
Is there a wording that you think is better? For the daily_tobegroupedby() function I have 
`start_day : int`
                    `Day of the start date  of the season.`

`start_month : int`
                    `Month of the start date of the season.`

Reply:
I think that makes sense. It's concise and precise. If users want to know more about what is meant by season, it's explained in the main description of the function. I would be tempted to keep the indications on syntax (1-31) and (1-12) because dates can be written in so many ways. I don't see any recommendation in the guidelines about that so... maybe not. I guess once there are examples, it's not necessary to put in the Parameters description